tests/sequence_sharding/data/db1.increment.sql

```diff
@@ -3,7 +3,8 @@ insert into t1 (uid,name) values (100003,'NR');
 update t1 set name = 'uxoKehvqWg' where `uid` = 100001;
 update t1 set name = 'bapYymrtfT' where name = 'igvApUx';
 insert into t2 (uid,name) values (200004,'CXDvoltoliUINgo'),(200005,'188689130');
-alter table t1 add column c int;
+alter table T1 add column c int;
```

Comment:
I have checked without code changing, this test will fail and `query-status` shows similar result as we expected